apps/qa/src/pages/ingest/helpers.py

```diff
@@ -0,0 +1,41 @@
+import shutil
+from pathlib import Path
+import streamlit as st
+from dcpy.utils import s3
+
+BUCKET = "edm-recipes"
+
+
+def archive_raw_data(dataset_name: str, version: str, uploaded_file, file_name: str):
+
+    base_path = Path(dataset_name)
```

Comment:
let's maybe make this `Path(".library") / "upload" / dataset_name`? That's a convention we've followed to some extent, `.library` is in our `.gitignore`, and it's nice to not just have things dumped in the current active folder